$NetBSD: patch-configure,v 1.1 2020/08/07 01:54:20 brook Exp $

Fix non-portable use of == with test.  Accepted by upstream.

--- configure.orig	2020-01-19 07:57:23.000000000 +0000
+++ configure
@@ -1828,7 +1828,7 @@ if test -z "${CURL_CONFIG}" ; then
   exit 1
 fi
 
-if test -n "${enable_debug}" && test "${enable_debug}" == "yes" ; then
+if test -n "${enable_debug}" && test "${enable_debug}" = "yes" ; then
  DEFINES=-DRCURL_DEBUG_MEMORY
 fi
 
